public/models.ts

```diff
@@ -0,0 +1,59 @@
+export interface AnimationSettings {
+  text: TextSettings;
+  background: BackgroundSettings;
+  speechBubble: SpeechBubbleSettings;
+  output: string;
+  textLocation: string;
+}
+
+export interface TextSettings {
+  fontFamily: string;
+  fontSize: number;
+  color: string;
+  bold: boolean;
+  italic: boolean;
+  highlightColor: string;
+  highlightRGB: string;
+}
+
+export interface BackgroundSettings {
+  color: string;
+  file: string;
+  type: string; // returns 'image' | 'video' | 'color'
```

Comment:
Use specific string types to limit values, i.e change this line to `type: 'image' | 'video' | 'color';`.

Reply:
done.